diff -ur redis-2.8.2.orig/runtest redis-2.8.2/runtest
--- redis-2.8.2.orig/runtest	2013-12-02 16:07:46.000000000 +0100
+++ redis-2.8.2/runtest	2013-12-04 21:57:48.613385391 +0100
@@ -1,10 +1,6 @@
 #!/bin/sh
-TCL_VERSIONS="8.5 8.6 8.7"
-TCLSH=""
-
-for VERSION in $TCL_VERSIONS; do
-	TCL=`which tclsh$VERSION 2>/dev/null` && TCLSH=$TCL
-done
+TCLSH="tclsh"
+TCL="tclsh"
 
 if [ -z $TCLSH ]
 then
diff -ur redis-2.8.2.orig/utils/speed-regression.tcl redis-2.8.2/utils/speed-regression.tcl
--- redis-2.8.2.orig/utils/speed-regression.tcl	2013-12-02 16:07:46.000000000 +0100
+++ redis-2.8.2/utils/speed-regression.tcl	2013-12-04 22:03:43.745711543 +0100
@@ -1,4 +1,4 @@
-#!/usr/bin/env tclsh8.5
+#!/usr/bin/tclsh
 # Copyright (C) 2011 Salvatore Sanfilippo
 # Released under the BSD license like Redis itself
 
