--- redis-8.0.1/src/Makefile~	2025-05-20 12:52:44.000000000 +0200
+++ redis-8.0.1/src/Makefile	2025-05-20 12:53:23.940397821 +0200
@@ -278,9 +278,8 @@ ifeq ($(MALLOC),tcmalloc_minimal)
 endif
 
 ifeq ($(MALLOC),jemalloc)
-	DEPENDENCY_TARGETS+= jemalloc
-	FINAL_CFLAGS+= -DUSE_JEMALLOC -I../deps/jemalloc/include
-	FINAL_LIBS := ../deps/jemalloc/lib/libjemalloc.a $(FINAL_LIBS)
+	FINAL_CFLAGS+= -DUSE_JEMALLOC $(shell $(PKG_CONFIG) --cflags jemalloc)
+	FINAL_LIBS := $(FINAL_LIBS) $(shell $(PKG_CONFIG) --libs jemalloc)
 endif
 
 # LIBSSL & LIBCRYPTO
